mbuild/formats/lammpsdata.py

```diff
@@ -736,6 +789,21 @@ def write_lammpsdata(
                             improper[3],
                         )
                     )
+            elif imp_dihedrals:
+                data.write("\nImpropers\n\n")
+                for i, improper in enumerate(imp_dihedrals):
+                    # The atoms were stored central-atom first so non-central atoms could be sorted
+                    # The atoms are written central-atom third in LAMMPS data file
```

Comment:
Can we add a comment like: `# This is correct for amber impropers even though the lammps documentation implies central-atom-first.`

Reply:
Sure.